brave/brave-core
# Option to disable autofill while in private browsing mode

Resolves https://github.com/brave/brave-browser/issues/9795

<img width="806" alt="image" src="https://github.com/brave/brave-core/assets/2965009/61a94b2f-013c-4f02-947f-09fc52cdc368">



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
credit cards:
- Fill and save info on regular profile https://fill.dev/form/credit-card-simple
- Disable the option in the settings
- Open it in Private profile and check.
- Saved info should not be substituted if the option is off.

Passwords:
- Login to Google and remember password in regular profile
- Open Google in Private Window
- It should not suggest saved password for login form in Private Window if the option disabled in the settings

Logins:
- Same as passwords but without saving password.
